browser-fetching/index.md

```diff
@@ -71,6 +71,10 @@ const fetchDog = () => fetch('https://dog.ceo/api/breeds/image/random')
   <img id="dog" style="width: 100%;" alt="Random dog image" src="https://via.placeholder.com/400" />
 </div>
 
+The HTTP request to fetch the dog data is reflected in the _Network_ tab, and the data from the fetched response can be inspected in the Console panel. 
+Below is an example of the data in the Web Developer tools.
+
+![Screenshot of dog data as an JavaScript object](../img/devtools-console-example.png)
```

Comment:
`as an JavaScript object` -> 'as a Javascript object'

Reply:
In the summary `The previous example only hint at the capabilities` -> `The previous example only hints at the capabilities`

Reply:
`a Fetch API for making a HTTP request to a web server` -> ` a Fetch API for making an HTTP request to a web server`
 
 This one is a little weird, but this sounds more natural to me since we pronounce the H normally.

Reply:
An "an" does seem more natural. I had always thought "a" should precede "h" since it is a consonant. According to a quick search, "h" is an exception:
Exceptions

> Use "an" before a slient or unsounded "h." Because the "h" does not have any phonetic representation or audible sound, the sound that follows the article is a vowel; consequently, "an" is used.
> 
>    an honorable peace
>    an honest error

Source: [Purdue Online Writing Lab](https://owl.purdue.edu/owl/general_writing/grammar/articles_a_versus_an.html)